client/src/components/LibraryFolder/TopToolbar/FolderTopBar.vue

```diff
@@ -0,0 +1,382 @@
+<template>
+    <div>
+        <div id="path-bar" />
+
+        <div class="form-inline d-flex align-items-center mb-2">
+            <a class="mr-1 btn btn-secondary" href="/library/list" data-toggle="tooltip" title="Go to first page">
```

Comment:
The href needs to be derived using `getAppRoot()` otherwise it'll fail in cases where the Galaxy root is not at `/`.